src/sage/tests/books/computational-mathematics-with-sagemath/recequadiff_doctest.py

```diff
@@ -56,9 +56,15 @@
 
 Sage example in ./recequadiff.tex, line 459::
 
-  sage: desolve(y == x*diff(y,x)-diff(y,x)^2, y,
-  ....:         contrib_ode=True, show_method=True)
-  [[y(x) == -_C^2 + _C*x, y(x) == 1/4*x^2], 'clairault']
+  sage: diffeq = y == x*diff(y,x)-diff(y,x)^2
+  sage: solution,method = desolve(diffeq,
+  ....:                           y,
+  ....:                           contrib_ode=True,
+  ....:                           show_method=True)
+  sage: solution
+  [y(x) == -_C^2 + _C*x, y(x) == 1/4*x^2]
+  sage: method in ["clairaut", "clairault"]
+  True
 
```

Comment:
Do you mean that the typo "clairault" is hard-coded into giac, you expect that it would be fixed in future?

Reply:
I'm only guessing that giac is to blame:

1. my giac is newer than sage's
2. both "clairaut" and "clairault" appear in the giac source code
3. there's no changelog for giac

But yes, my guess is that "clairault" was hard-coded somewhere and now it's "clairaut".